02_knight_and_princess/tamara_gadieva/edge_func.cpp

```diff
@@ -0,0 +1,9 @@
+#include "graph_generate.hpp"
+
+std::string Edge::json_string() const {
+  std::string result_edge_inf =
+      "\t\t{ \"id\": " + std::to_string(id) + ", \"vertex_ids\": [" +
+      std::to_string(vertex_id1) + ", " + std::to_string(vertex_id2) + "] }";
+
+  return result_edge_inf;
```

Comment:
Тебе не нужна переменная только для возврата:
```suggestion
  return "\t\t{ \"id\": " + std::to_string(id) + ", \"vertex_ids\": [" +
      std::to_string(vertex_id1) + ", " + std::to_string(vertex_id2) + "] }";
```